### PR TITLE
1st module task

### PR DESCRIPTION
**Pull request by OlenaEpam**

### CODE REVIEW

File: hw_1/tasks.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+# Import mean method from statistic module
+from statistics import mean
+# Import module which has sample method in it
+import random
+
+# Task1. Create list of 100 random numbers from 0 to 1000
+"""Create a variable random_list, call for a method sample from random module which requires range 
+and number of required numbers. The method will return a list"""
+random_list = random.sample(range(0, 1000), 100)
```
Comment (BulbaYim) [inline hw_1/tasks.py]: range() doesn't include the last element, sample() chooses between 0 and 999

File: hw_1/tasks.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    else:
+        odd_numbers.append(i)
+print(f"This is a list with even numbers: {even_numbers}")
+print(f"This is a list with odd numbers: {odd_numbers}")
+
+# using mean method let's get average from list of numbers. In case of empty list, print error from Except clause
+try:
+    average_of_even = mean(even_numbers)
+    print(f"Average of even numbers is: {average_of_even}")
+except ZeroDivisionError:
```
Comment (BulbaYim) [inline hw_1/tasks.py]: ZeroDivisionError occurs when we try to divide a number by 0, but in this case mean() is called
You can check that list is not empty, before using mean()

File: hw_1/tasks.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+# and which are odd(with residual)
+for i in sorted_list:  # for every number in a list
+    if i % 2 == 0:  # if number is divided without residual
+        even_numbers.append(i)  # put this number in a list of even numbers
+    else:
+        odd_numbers.append(i)
+print(f"This is a list with even numbers: {even_numbers}")
+print(f"This is a list with odd numbers: {odd_numbers}")
+
+# using mean method let's get average from list of numbers. In case of empty list, print error from Except clause
```
Comment (BulbaYim) [inline hw_1/tasks.py]: It's better not to use too long comments https://peps.python.org/pep-0008/#maximum-line-length

**Review by BulbaYim — CHANGES_REQUESTED** [review]

File: hw_1/tasks.py (hunk 1 above)
Comment (OlenaEpam) [reply hw_1/tasks.py]: Thanks, corrected to range(0,1001)

File: hw_1/tasks.py (hunk 2 above)
Comment (OlenaEpam) [reply hw_1/tasks.py]: Corrected, removed Except and introduced check for empty list before using mean.

File: hw_1/tasks.py (hunk 3 above)
Comment (OlenaEpam) [reply hw_1/tasks.py]: Corrected, shortened the comment. Thank you

**Review by BulbaYim — APPROVED** [review]

**Pull request merged**